app/controllers/projects_controller.rb

```diff
@@ -217,4 +218,8 @@ def find_project_by_id
     def redirect_to_slug
       redirect_to @project if @project
     end
+
+    def username_arr
+      @una = User.pluck(:name).compact.to_json
```

Comment:
`@una` what is his meaning?

Reply:
username array  :smile:  i ll edit it to usernamearray